From ba0760ba526cad0315b2aad1484e170e699f1818 Mon Sep 17 00:00:00 2001
From: Gustavo Jose de Sousa <gustavo.sousa@intel.com>
Date: Wed, 16 Nov 2016 13:23:49 -0200
Subject: [PATCH] Fix @PKG_NAME@_INCLUDE_DIR variable

The previous approach hardcoded the path to the include directory and cross
compilation using a sysroot directory would fail. This patch takes the
resulting config files generated from other ROS packages.
---
 config.cmake.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config.cmake.in b/config.cmake.in
index b62ab64..7ae9af1 100644
--- a/config.cmake.in
+++ b/config.cmake.in
@@ -3,7 +3,8 @@ if (@PKG_NAME@_CONFIG_INCLUDED)
 endif()
 set(@PKG_NAME@_CONFIG_INCLUDED TRUE)
 
-set(@PKG_NAME@_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
+get_filename_component(include "${@PKG_NAME@_DIR}/../../../include" ABSOLUTE))
+set(@PKG_NAME@_INCLUDE_DIRS ${include})
 set(@PKG_NAME@_DIALECTS @PKG_MAVLINK_DIALECTS@)
 set(@PKG2_NAME@_DIALECTS @PKG2_MAVLINK_DIALECTS@)
 
-- 
2.10.2

